api2_deprecated/index.html

```diff
@@ -1,19 +1,23 @@
 ---
```

Comment:
Can we just delete this file? URL has changed so users can't find it anyway + if we need it we can look through source control